src/universal/modules/meeting/helpers/electFacilitatorIfNone.js

```diff
@@ -0,0 +1,29 @@
+import {cashay} from 'cashay';
+import {showInfo} from 'universal/modules/toast/ducks/toastDuck';
+
+const electFacilitatorIfNone = (nextProps, oldMembers) => {
+  const {dispatch, team: {activeFacilitator}, members} = nextProps;
+  if (!activeFacilitator) return;
```

Comment:
Does this mean that the meeting hasn't started yet and a facilitator has never been elected?